app/furniture/marketplace/stripe_events_controller.rb

```diff
@@ -21,7 +21,7 @@ def create
         latest_charge = Stripe::Charge.retrieve(payment_intent.latest_charge, api_key: marketplace.stripe_api_key)
         balance_transaction = Stripe::BalanceTransaction.retrieve(latest_charge.balance_transaction, api_key: marketplace.stripe_api_key)
 
-        order.update(status: :paid, placed_at: DateTime.now)
+        order.update(status: :paid, placed_at: DateTime.now, payment_processor_fee_cents: balance_transaction.fee)
```

Comment:
Not from this PR, but I'm noticing that we do this `.update` and don't seem to be checking for success, opening us up to the possibility that this update will fail, the order won't be marked as `paid`, but notifications and other parts of the "order pipeline" would continue as if everything was fine.

One quick fix would be to throw a `!` onto the `order.update` call.

Reply:
Good point! Will do!